erpnext/stock/doctype/packed_item/packed_item.py

```diff
@@ -103,6 +106,45 @@ def cleanup_packing_list(doc, parent_items):
 		if d not in delete_list:
 			doc.append("packed_items", d)
 
+def update_product_bundle_price(doc, parent_items):
+	"""Updates the prices of Product Bundles based on the rates of the Items in the bundle."""
+
+	parent_items_index = 0
+	bundle_price = 0
+	parent_items_doctype = doc.items[0].doctype
```

Comment:
Will this break if `doc.items` is empty?

Reply:
Theoretically, yes, but why would anyone save an Invoice/Order that doesn't have any Items?

Reply:
Hmm, but better return if there are no items

Reply:
Done.